test/e2e/framework.go

```diff
@@ -705,14 +712,69 @@ func (data *TestData) enableAntreaFlowExporter(ipfixCollector string) error {
 	return data.mutateAntreaConfigMap(nil, ac, false, true)
 }
 
-func (data *TestData) disableAntreaFlowExporter() error {
-	ac := func(config *agentconfig.AgentConfig) {
-		config.FeatureGates["FlowExporter"] = false
+// deployFlowVisibilityClickHouse deploys ClickHouse operator and DB.
+func (data *TestData) deployFlowVisibilityClickHouse() (string, error) {
+	err := data.CreateNamespace(flowVisibilityNamespace, nil)
+	if err != nil {
+		return "", err
 	}
-	return data.mutateAntreaConfigMap(nil, ac, false, true)
+
+	rc, _, _, err := data.provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl apply -f %s", chOperatorYML))
+	if err != nil || rc != 0 {
+		return "", fmt.Errorf("error when deploying the ClickHouse Operator YML; %s not available on the control-plane Node", chOperatorYML)
+	}
+	if err := wait.Poll(2*time.Second, 10*time.Second, func() (bool, error) {
+		rc, stdout, stderr, err := data.provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl apply -f %s", flowVisibilityYML))
+		if err != nil || rc != 0 {
+			// ClickHouseInstallation CRD from ClickHouse Operator install bundle applied soon before
+			// applying CR. Sometimes apiserver validation fails to recognize resource of
+			// kind: ClickHouseInstallation. Retry in such scenario.
+			if strings.Contains(stderr, "ClickHouseInstallation") || strings.Contains(stdout, "ClickHouseInstallation") {
+				return false, nil
+			}
+			return false, fmt.Errorf("error when deploying the flow visibility YML %s: %s, %s, %v", flowVisibilityYML, stdout, stderr, err)
+		}
+		return true, nil
+	}); err != nil {
+		return "", err
+	}
+
+	// check for clickhouse pod Ready. Wait for 2x timeout as ch operator needs to be running first to handle chi
+	if err = data.podWaitForReady(2*defaultTimeout, flowVisibilityCHPodName, flowVisibilityNamespace); err != nil {
+		return "", err
+	}
+
+	// check clickhouse service http port for service connectivity
+	chSvc, err := data.GetService("flow-visibility", "clickhouse-clickhouse")
+	if err != nil {
+		return "", err
+	}
+	if err := wait.Poll(defaultInterval, defaultTimeout, func() (bool, error) {
+		rc, stdout, stderr, err := testData.RunCommandOnNode(controlPlaneNodeName(),
+			fmt.Sprintf("curl -Ss %s:%s", chSvc.Spec.ClusterIP, clickHouseHTTPPort))
+		if rc != 0 || err != nil {
+			log.Infof("Error curl clickhouse service: %s", strings.Trim(stderr, "\n"))
```

Comment:
```suggestion
			log.Infof("Failed to curl clickhouse Service: %s", strings.Trim(stderr, "\n"))
```

do we use `log.Info` in other places?

Reply:
done. it's used in several other places under the same file. for example:
`func killProcessAndCollectCovFiles`: `log.Infof("Sending SIGINT to '%s'", processName)`
`func DeleteNamespace`: `log.Infof("Deleting Namespace %s took %v", namespace, time.Since(startTime))`